1/backend/src/todos.spec.js

```diff
@@ -122,31 +171,23 @@ describe('Updates Todo Item', () => {
 
 describe('Delete Todo', () => {
     it("deletes Todo", async () => {
-        const todo = await mutate({mutation: DELETE_TODO, variables: {id: 2}})
+        const todo = await mutate({mutation: DELETE_TODO, variables: {id: test_id}})
         expect(todo.data).toMatchObject(
             {"deleteTodo": true}
         )
     })
     it("deletes Todo with wrong ID", async () => {
         const todo = await mutate({mutation: DELETE_TODO, variables: {id: 12}})
         expect(todo.data).toMatchObject(
-            {"deleteTodo": false}
+            {"deleteTodo": true}
```

Comment:
This is why I don't like `Boolean` as return type. What on earth is the meaning of `true`? :thinking: